src/v2/view-builder/views/email/EnrollAuthenticatorEmailView.js

```diff
@@ -10,7 +10,8 @@ const Body = BaseAuthenticatorEmailForm.extend(
     initialize() {
       BaseAuthenticatorEmailForm.prototype.initialize.apply(this, arguments);
 
-      const email = this.options.appState.get('user')?.identifier || {};
+      const email = this.options.appState.get('user')?.profile?.email || (this.options.appState.get('user')?.identifier
```

Comment:
```suggestion
      const email = this.options.appState.get('user')?.profile?.email || this.options.appState.get('user')?.identifier || {}
```
wrap is not needed at the end